src/hub/atlas.js

```diff
@@ -421,6 +421,79 @@ class AtlasClient {
         return users;
     }
 
+    /**
+     * Update an existing users data
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint PATCH method
+     */
+    async patchUser(data) {
+        const op = { method: "PATCH", body: { ...data } };
+        try {
+            const resp = await this.fetch("/v1/user/" + data.id, op);
+            if(!resp.ok) {
+                const e = new errors.ProtocolError(resp.status, respContent);
+                e.message = `${urn} (${text})`;
+                throw e;
+            }
+        } catch (err) {
+            throw err;
+        }
+    }
+
+    /**
+     * Add a new user to the existing user set
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint POST method
+     */
+    async postUser(data) {
+        const op = { method: "POST", body: { ...data } };
+        try {
+            const resp = await this.fetch("/v1/user/", op);
+            if(!resp.ok) {
+                const e = new errors.ProtocolError(resp.status, respContent);
+                e.message = `${urn} (${text})`;
+                throw e;
+            }
+            return true;
+        } catch (err) {
+            throw err;
+        }
+    }
+
+    /**
+     * Register a new user with atlas
+     *
+     * @param {Object} body - A user data object in the form:
+     *  {   
+     *      captcha, 
+     *      phone, 
+     *      email, 
+     *      fullname, 
+     *      tag_slug, 
+     *      password, 
+     *      org_name, 
+     *      org_slug 
+     * }
+     * All fields are required except phone
+     * 
+     * Use the recaptcha site key 6LcYOD8UAAAAAK8h67aZdxUJqgDPNpgIEDFgopil.
```

Comment:
Same as, before, just same something, like they need to contact support@forsta.io for access.